core/object/object.cpp

```diff
@@ -492,14 +492,22 @@ void Object::get_property_list(List<PropertyInfo> *p_list, bool p_reversed) cons
 			ClassDB::get_property_list(current_extension->class_name, p_list, true, this);
 
 			if (current_extension->get_property_list) {
-				uint32_t pcount;
-				const GDExtensionPropertyInfo *pinfo = current_extension->get_property_list(_extension_instance, &pcount);
-				for (uint32_t i = 0; i < pcount; i++) {
-					p_list->push_back(PropertyInfo(pinfo[i]));
-				}
-				if (current_extension->free_property_list) {
-					current_extension->free_property_list(_extension_instance, pinfo);
+#ifdef TOOLS_ENABLED
+				// If this is a placeholder, we can't call into the GDExtension on the parent class,
+				// because we don't have a real instance of the class to give it.
+				if (!_extension->is_placeholder) {
```

Comment:
`unlikely()` recommended here too

Reply:
I think you probably mean `likely()` in this case? This is checking that it's _not_ a placeholder, meaning this branch would be taken by the non-placeholder case. Added as `likely()` in my latest push